sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/cryptography_client.hpp

```diff
@@ -42,7 +42,6 @@ namespace Azure {
    *
    */
   class CryptographyClient final {
-  protected:
```

Comment:
I don't think we can do this (breaking change).

Reply:
It's not a breaking change - the CryptographyClient is declared as final, so there can never be any classes derived from CryptographyClient.